devdox/app/services/repository.py

```
@@ -158,6 +160,32 @@ async def retrieve_repo_by_id(repo_repository_instance: RepoRepository, id):
+            # GitHub returns AuthenticatedUser object
+            return {
+                "repo_author_name": user.name or user.login,
+                "repo_author_email": user.email or f"{user.login}@users.noreply.github.com"
+            }
+        elif git_provider.lower() == "gitlab":
+            # GitLab returns dict
+            return {
+                "repo_author_name": user.get("name") or user.get("username"),
+                "repo_author_email": user.get("email") or f"{user.get('username')}@gitlab.local"
+            }
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)